package.json

```diff
@@ -24,6 +24,9 @@
     "babel-plugin-lodash": "^3.3.4",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.18"
   },
+  "optionalDependencies": {
+    "@babel/preset-typescript": "^7.1.0"
```

Comment:
[`optionalDependencies`](https://docs.npmjs.com/files/package.json#optionaldependencies) is used for dependencies that may fail to compile or function on some platforms and should have a fallback, rather than dependencies that may or may not be used by the package, so it doesn't make sense to use here. Can you put this in `dependencies` instead?

Reply:
oooh didn't know, thanks for the info!